spec/features/idv/doc_auth/verify_info_step_spec.rb

```diff
@@ -271,8 +271,8 @@
       {
         trace_id: anything,
         threatmetrix_session_id: anything,
-        request_ip: kind_of(String),
         double_address_verification: false,
+        request_ip: kind_of(String),
       }
```

Comment:
One less file change if this is restored- but no harm is this edit

Reply:
good catch, i'll change it